clang/include/clang/CConv/ProgramInfo.h

```diff
@@ -29,6 +29,10 @@ class ProgramInfo;
 
 class ProgramInfo {
 public:
+  typedef std::map<unsigned int, std::string> CallTypeParamBindingsType;
+  typedef std::map<PersistentSourceLoc, CallTypeParamBindingsType>
+      TypeParamBindingsType;
```

Comment:
See comment above. Reads oddly to me to have the typename end with `Type`; change to `T` instead (or whatever the clang style guide says)?